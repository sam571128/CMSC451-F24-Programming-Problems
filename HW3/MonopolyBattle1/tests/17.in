23 529 1000
14 13
7 17
14 1
12 15
20 12
14 9
14 3
18 1
21 4
12 1
13 5
17 7
23 12
22 2
11 13
21 16
5 17
1 20
3 8
15 10
18 13
6 6
12 10
16 4
4 21
20 10
5 15
13 12
22 21
19 22
18 11
16 23
16 13
11 2
21 10
1 17
22 12
14 22
8 20
4 15
20 9
4 10
9 20
1 1
18 9
8 14
17 19
19 9
5 14
5 16
9 12
21 23
9 6
16 22
23 5
7 1
5 9
11 3
8 21
23 6
12 6
23 2
15 18
5 11
23 7
2 12
11 20
19 3
15 15
18 14
22 23
3 14
10 3
17 9
8 23
3 1
11 17
18 6
11 18
3 17
20 5
3 9
22 15
19 15
22 5
10 12
4 6
20 3
10 6
9 3
1 19
12 8
12 16
19 19
10 19
9 17
16 14
15 13
3 11
13 13
7 11
10 1
8 8
23 4
1 13
6 21
20 21
17 18
12 11
17 22
7 19
21 13
21 17
22 9
12 19
21 7
6 13
13 15
22 3
5 12
12 23
21 18
4 20
2 8
10 2
13 3
16 15
10 10
19 23
7 6
8 19
2 10
2 19
9 19
9 13
10 7
16 16
19 7
15 20
11 16
7 13
18 8
23 18
5 2
4 1
2 18
15 6
21 1
21 21
4 22
18 15
19 11
23 19
17 12
14 2
9 23
9 10
20 2
9 15
12 22
16 9
23 9
14 21
22 18
7 22
13 11
19 20
17 6
11 9
17 8
6 5
13 7
17 14
14 19
2 17
4 3
2 6
10 14
13 9
16 19
13 22
8 12
21 6
19 21
11 12
2 22
20 4
1 9
3 18
11 21
9 14
16 2
23 23
7 15
3 6
22 8
17 15
22 6
9 9
9 22
11 5
7 7
15 14
18 10
8 7
6 8
8 16
10 21
4 12
4 11
20 19
19 17
21 2
22 10
18 23
9 18
5 4
12 12
6 10
17 2
2 11
8 22
5 10
7 12
4 9
16 5
18 3
8 11
8 18
16 17
23 10
2 20
3 5
4 4
12 17
2 15
3 12
1 21
18 20
15 8
13 19
19 1
6 12
9 1
13 8
18 4
10 11
11 7
13 20
20 20
23 3
10 23
16 1
18 16
16 10
18 21
7 4
12 21
1 15
11 14
17 10
11 15
5 19
14 20
19 4
1 23
4 14
6 14
7 10
2 7
10 18
22 13
20 6
5 7
6 3
3 10
11 11
1 11
23 21
21 11
15 7
4 7
20 14
16 11
5 23
13 6
1 7
10 20
19 5
12 2
15 17
3 13
18 5
14 4
14 16
4 18
20 18
2 14
16 20
23 1
9 5
14 8
7 5
20 13
1 4
17 20
12 18
17 13
7 16
17 4
10 22
4 2
13 10
2 3
22 11
8 6
19 14
1 6
20 8
22 1
6 23
23 13
12 20
7 23
4 13
10 15
13 21
7 21
6 2
4 23
6 20
19 2
20 22
20 7
4 17
3 3
23 8
5 3
13 14
18 12
2 9
15 21
18 17
9 7
15 2
6 1
6 9
10 13
14 5
7 8
1 18
5 21
16 12
5 6
23 16
11 8
6 19
20 15
8 9
11 22
18 2
3 16
8 10
18 19
14 6
19 8
16 21
21 20
3 19
13 2
3 15
8 5
11 6
17 23
9 2
5 13
22 7
21 3
19 10
17 11
1 16
1 22
16 3
4 8
4 5
13 17
19 18
20 17
5 8
18 7
2 1
6 16
2 13
10 8
12 3
8 1
15 23
20 11
12 7
11 19
8 2
15 12
12 14
23 17
20 1
14 17
1 12
2 4
17 16
2 23
22 14
14 12
7 14
4 19
6 18
22 20
22 19
2 21
9 16
14 14
12 4
14 7
12 5
8 3
10 16
22 17
23 15
8 4
2 16
20 23
15 5
10 17
15 3
13 23
7 2
19 16
9 8
23 11
7 20
9 21
3 20
1 2
19 12
6 7
3 4
17 1
21 9
3 21
21 22
21 19
16 7
7 3
4 16
12 9
11 4
9 11
21 12
10 5
21 8
17 5
15 22
10 4
19 6
1 5
17 21
10 9
21 15
13 16
3 2
15 16
16 8
1 10
14 15
19 13
13 18
5 1
14 11
7 9
21 14
5 18
17 3
5 5
14 18
23 14
7 18
6 17
18 18
6 15
21 5
23 20
11 23
3 22
3 7
15 9
1 14
11 1
18 22
15 1
12 13
1 3
3 23
15 19
5 22
13 1
15 4
22 4
6 22
2 5
14 23
9 4
23 22
16 6
8 13
17 17
15 11
1 8
22 16
16 18
14 10
8 17
6 4
20 16
8 15
6 11
11 10
5 20
22 22
13 4
2 2
