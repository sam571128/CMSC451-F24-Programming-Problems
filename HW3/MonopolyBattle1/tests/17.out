43 47 49 47 47 53 36 49 43 20 63 43 47 49 43 47 36 63 47 18 47 47 47 